documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -429,42 +457,49 @@ log.flush.interval.messages=100000
+Followers do not normally serve clients, though xref:type-Rack-reference[`rack` configuration] allows a consumer to consume messages from the closest replica when a Kafka cluster spans multiple datacenters.
+Followers operate only to replicate messages from the partition leader and allow recovery should the leader fail.
+Recovery requires an in-sync follower. Followers stay in sync by sending fetch requests to the leader, which returns messages to the follower in order.
+The follower is considered to be in sync if it has caught up with the most recently committed message on the leader.
+The leader checks this by looking at the last offset requested by the follower.
+An out-of-sync follower is usually not eligible as a leader should the current leader fail, unless xref:con-broker-config-properties-unclean-{context}[unclean leader election is allowed].
 
 You can adjust the lag time before a follower is considered out of sync:
 
 [source,env]
 ----
 # ...
-replica.lag.time.max.ms
+replica.lag.time.max.ms=30000
 # ...
 ----
 
-This is the latency you will accept between the leader and followers that are considered in sync. You can decrease the time to reduce the lag, but by doing so you might increase the number of followers that fall out of sync. The right value depends on both network latency and broker disk bandwidth.
-
-NOTE: With a high lag time data might be lost in certain circumstances.
-Suppose your partition leader fails and followers have not caught up with the latest messages.
-Messages not flushed in the period between a follower sending its next request for messages (while it is still considered in sync) and transition to a new leader might be lost.
+Lag time puts an upper limit on the time to replicate a message to all in-sync replicas and how long a follower can remain in sync.
+If a follower fails to make a fetch request and catch up with the latest message within the specified lag time, it is removed from in-sync replicas.
+You can reduce the lag time to detect failed replicas sooner, but by doing so you might increase the number of followers that fall out of sync needlessly.
+The right lag time value depends on both network latency and broker disk bandwidth.
 
-When the leader partition is no longer available, one of the in-sync replicas is chosen as the new leader.
+When a leader partition is no longer available, one of the in-sync replicas is chosen as the new leader.
 
-The first broker in a partition’s list of replicas is known as the preferred replica.  Kafka tries to ensure that, on average, each broker is the _preferred_ replica for a similar number of partitions.   If a leader fails, this affects the balance of a Kafka cluster (as does the assignment of partition replicas to brokers).
+The first broker in a partition’s list of replicas is known as the preferred replica.
+Kafka tries to ensure that, on average, each broker is the _preferred_ replica for a similar number of partitions.
```

Comment:
Let's drop this, and it's a bit more complex than this. We can use CC to ensure that, _when_ every partition is lead by its preferred replica, the cluster is approximately optimally balanced. CC does this by figuring out which broker is the best to be the preferred leader for each partition, and reassigning partitions/changing the preferred leader to achieve this. Kafka doesn't do this on its own (all it does try to assign new partitions "evenly", which is not the same thing because different partitions can impart very different loads on brokers). 

_If_ the preferred leaders are optimally arranged then the Kafka feature of ensuring that the current leader is the preferred leader (as much as possible) is useful. If they preferred leader are badly arranged then the Kafka feature achieves nothing.

```suggestion
```